cmake/sca/polyspace/sca.cmake

```diff
@@ -105,10 +103,12 @@ add_custom_target(polyspace-analyze ALL
     -options-file ${POLYSPACE_OPTIONS_FILE_BASE}
     ${POLYSPACE_PROG_VERSION}
     ${POLYSPACE_OPTIONS_ZEPHYR}
+    ${POLYSPACE_OPTIONS_ZEPHYR_ONLYAPP}
     ${POLYSPACE_OPTIONS_FILE}
     ${POLYSPACE_OPTIONS}
-      || ${CMAKE_COMMAND} -E true # allow to continue processing results
+    || ${CMAKE_COMMAND} -E true # when there are no sources...no easy way to skip this target with cmake syntax
```

Comment:
but there are sources.

as you have just experienced in https://github.com/zephyrproject-rtos/zephyr/pull/98872#discussion_r2511001317 then just trying to get CMake / build to proceed and not understading what actually happens is not fixing anything.

Instead it will lead to hard to understand / debug issues.

Now all errors raised by `${POLYSPACE_EXE}` will be silently ignored.